music_this_week_app/templates/music_this_week/setup.html

```diff
@@ -12,23 +12,75 @@
         <p><a href="{{ profile }}" target="_blank">{{ username }}</a>&nbsp; &nbsp;&nbsp;<a href="/login/?retry=true">Switch Users</a></p>
         <p><span style="text-decoration: underline;">Search Arguments:</span></p>
         <form action="/search" id="form">
-            <p><strong>Location:</strong> <input id = "loc_field" type="text" name="location" value="San Francisco"></p>
+            <p id = 'location_options'><strong>Location:</strong> <input id = "loc_field" type="text" name="location" value="{{location}}"> 
```

Comment:
This is a slightly more complex HTML structure than is used elsewhere in the doc. Can you please update the indentation to make the structure more legible/obvious?